lib/components/show_help.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:maui/components/topic_page_view.dart';
+
+class ShowHelpDrawer extends StatelessWidget {
```

Comment:
file_name.dart and FileName class should match. In this case show_help.dart is the file name, but ShowHelpDrawer is the class name.